tests/opt-testcases/tls13-kex-modes.sh

```diff
@@ -17,6 +17,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+P_SRV_PSK_LIST="psk_list=abc,dead,def,beef,Client_identity,6162636465666768696a6b6c6d6e6f70"
```

Comment:
To exercise in different ways the PSK look-up code on server side, what about to have 3 lists:
1) One with the Client_identity PSK being first in the list
2) One with the Client_identity PSK being last in the list (as the current one)
3) One with the Client_identity PSK being in the middle
and to use them in a randomly manner in the various tests.